src/main/scala/scala/tools/partest/nest/Runner.scala

```diff
@@ -108,7 +108,9 @@ class Runner(val testFile: File, val suiteRunner: SuiteRunner) {
       "-d",
       outDir.getAbsolutePath,
       "-classpath",
-      joinPaths(outDir :: testClassPath)
+      joinPaths(outDir :: testClassPath),
+      "-J-Duser.language=en",
+      "-J-Duser.country=us"
```

Comment:
I think this needs to be `US`. Language codes are lowercase, country codes are upper case (like in `propertyOptions` in the same file)